packages/manager/apps/dedicated/client/app/ip/filters/availableIpText/index.js

```diff
@@ -0,0 +1,35 @@
+export default /* @ngInject */ ($translate) =>
```

Comment:
- I've added `/* @ngInject */`
- Except from that, nothing has been done here except moving the file